Adding steps for updating license (#505)
hashicorp/terraform-website: content/source/docs/enterprise/private/updating-license.html.md

---
layout: "enterprise2"
page_title: "Updating License Key"

> sidebar_current: "docs-enterprise2-private-installer-rhel"

Reviewer: This needs to be changed to something unique.

Author: Copypasta error, fixed!

---

# Updating License Key

> This guide is for updating the License file for an installation of Private Terraform Enterprise.

Reviewer: "license" and "license key" should be lowercase. 

This intro sentence should include a brief word about when an admin would want to/be able to update a license file.

Author: Fixed! 👍

If you are missing the new License Key, please talk to to your **Solutions Engineer** (pre-sales, POC or trial) or **Technical Account Manager** (existing customers).

## Requirements

> * shell access to the VM running Private Terraform Enterprise

Reviewer: capital "Shell"

Author: Done! 👍

* The license file (with the extension `.rli`)

## Steps

> ### If using Replicated > `Version`

Reviewer: - Is this going to be replaced by a version number? 
- Would it make sense to use PTFE release numbers instead of Replicated versions, or is the Replicated version exposed clearly to admins? (I haven't yet installed a full PTFE instance, so I haven't used the Replicated dashboard.)

Author: Replicated uses a different release pattern than PTFE, the version is exposed either via the CLI or the replicated UI. 

I do need to find out the version number where this changed however...

1. Copy the new license file to the PTFE instance (for example with `scp`)
2. `ssh` to the PTFE instance
3. Load the license, by running `cat <license_file> | replicatedctl license-load`
4. In the PTFE Admin Console, select the "View License" menu under the gear icon (or navigate directly to <host>:8800/license.
5. Click the "Sync License" button.
6. Go to the dashboard using the Dashboard menu.
7. Restart the PTFE application in the dashboard.

### If using Replicated < `Version`

1. Copy the new license file to the PTFE instance (for example with `scp`)
2. `ssh` to the PTFE instance
3. Set the license file location with the command line: `replicatedctl params set LicenseFileLocation --value '/full/path/to/license.rli`
4. Stop PTFE in the Web UI under `<host>:8800/`

> 4. Stop Replicated:

Reviewer: For these code blocks nested inside a list item:

- Add a blank line before and after the code block.
- Indent the entire code block 4 spaces, including the triple-backtick delimiters. 
- Use `shell` as the language, and prefix the lines with `$ `.

Author: Fixed 👍

---
layout: "enterprise2"
page_title: "Updating License Key"
sidebar_current: "docs-enterprise2-private-installer-license"
---

# Updating License Key

> This guide is for updating the license file for an installation of Private Terraform Enterprise. You will want to do before a license expiry, such as after a commercial renewal.

Reviewer: ```suggestion
After renewing your commercial license for Private Terraform Enterprise (PTFE), follow the instructions below to update the license file for your installation.
```

If you are missing the new license Key, please talk to to your **Solutions Engineer** (pre-sales, POC or trial) or **Technical Account Manager** (existing customers).

## Requirements

* Shell access to the VM running Private Terraform Enterprise
* The license file (with the extension `.rli`)

## Steps

### Online Installation

> > For the online installation mode, it is not necessary for a customer to manually load a new license file to their PTFE servers. That is done for them by the "Sync License" button.

Reviewer: ```suggestion
For the online installation mode, it is not necessary for a customer to manually load a new license file to their PTFE servers. That is done for them by the "Sync License" button.
```

1. In the PTFE Admin Console, select the "View License" menu under the gear icon (or navigate directly to <ptfe-hostname>:8800/license.
1. Click the "Sync License" button.
1. Go to the dashboard using the Dashboard menu.
1. Restart the PTFE application in the dashboard.

### Airgap installation

> > For an Airgap installation, a new license file will have to be provided by the customer's Technical Account Manager or Solutions Engineer.

Reviewer: ```suggestion
For an Airgap installation, a new license file will have to be provided by the customer's Technical Account Manager or Solutions Engineer.
```
